### DISCUSSION TITLE
Update not

### DISCUSSION THREAD

**Discussion by ycatois** [dbeaver/dbeaver#20495] [Q&A] [answered]
### Description

The automatic update is done but the version remains in 23.0.1??
Even if I download the latest version (23.1.1) and install it, the application remains in 23.0.1
However, no error message, everything is done as if everything was fine.

### DBeaver Version

Community edition 23.0.1

### Operating System

Windows 10

**Comment by LonwoLonwo** [comment, marked as answer]
Hello @ycatois 

Did you choose the installation for one person or all? You may have two DBeaver applications on your PC.

**Comment by ycatois** [comment]
Hi,

Yes, that was the cause. Thank you for helping me :)

The ticket is solved.

Yann Catois
